README.md

```diff
@@ -46,29 +56,29 @@
 3. Check if ANT is already installed or not by typing "ant -version". If ant is installed, you will get output. If not, you 
    will get 'command not found'. If it is not already installed install using the steps below.
 
-a. Check if brew is installed or not by typing "brew help". If brew is installed, you will get output. If not, you will get 
+  a. Check if brew is installed or not by typing "brew help". If brew is installed, you will get output. If not, you will get 
    'command not found'. If it is not already installed install using
    ```bash
    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
    ```
 
-b. Install ANT using
+  b. Install ANT using
    ```bash
    $ brew install ant
    ```  
 
 4. Set ANT_HOME environment variable. 
 
-a. In your home folder, open the .bash_profile file in the root directory.
+  a. In your home folder, open the .bash_profile file in the root directory.
 
-b. Add the following lines to the file, substituting the directory where you installed Ant:
+  b. Add the following lines to the file, substituting the directory where you installed Ant:
    ```bash
    ANT_HOME=/apache-install-dir/apache-ant-version
    PATH=$PATH:$HOME/bin:$ANT_HOME/bin
```

Comment:
Because it may not be installed this way. It's possible that user installs ant just in, say, /usr/local/apache, ignoring version.  install dir should show explicit path to parent directory of bin, lib, etc - it may or may not contain version subdirs.  But I don't insist, it's just my take.